docs/Available-Symbols-Generators.md

```diff
@@ -106,12 +105,16 @@ In this sample three symbols are defined:
 ## Coalesce
 Behaves like the C# `??` operator.
 
+Note: as in C# `??` operator, empty string will be accepted as the valid value and returned. 
+The typical use of this generator is to check if the parameter was provided by user, otherwise set fallback generated value.
+
 #### Parameters
 
-| Name     |Data Type| Description   |
-|----------|------|---------------|
-|sourceVariableName|string|source of the data|
-|fallbackVariableName|string|alternate source of data|
+|Name|Data Type|Description|Mandatory|
+|---|---|---|---|
+|`sourceVariableName`|`string`|the symbol name which is a primary source of data (left operand of `coalesce`)|yes|
+|`fallbackVariableName`|`string`|the symbol name which is an alternate source of data(right operand of `coalesce`)|yes|
```

Comment:
Was this mandatory in previous versions?
Though coalesce without fallback value doesn't make much sense - so regression might be fine. But still probably worth checking if there are any templates with coalesce without fallbackVariableName.